contentcuration/contentcuration/settings.py

```diff
@@ -331,3 +331,9 @@
 
 # Used as the default parent to collect orphan nodes
 ORPHANAGE_ROOT_ID = "00000000000000000000000000000000"
+
+# IMPORTANT: Deleted chefs should not be in the orhpanage becuase this can lead to very large and painful resorts
+# of the tree. This tree is special in that it should always be accessed inside a disable_mptt_updates code block,
+# so we must be very careful to limit code that touches this tree and to carefully check code that does. If we
+# do choose to implement restore of old chefs, we will need to ensure moving nodes does not cause a tree sort.
+DELETED_CHEFS_ROOT_ID = "00000000000000000000000000000000"
```

Comment:
This looks really similar to the orphanage root ID. Won't they just end up in the same tree?

Reply:
Gack! This is what reviews are for. 🤦‍♂️ In previous versions of the code, I had a 1 at the end, but maybe I went back one step too far with an undo or something. I guess we could use all ones as well to make it more obvious.